### DISCUSSION TITLE
Melee quickswitch

### DISCUSSION THREAD

**Discussion by AsiaGuy** [IceHacks/SurvivCheatInjector#747] [Ideas]
When someone come close enough to you, auto switch to the melee to kim him. And when he reach out of the melee range, auto switch back to the previous gun
Note: the previous hack don't have the 2nd feature

**Comment by AsiaGuy** [comment]
Also please resend the discord invition link

**Comment by iBLiSSIN** [comment]
> When someone come close enough to you, auto switch to the melee to kim him. And when he reach out of the melee range, auto switch back to the previous gun
> Note: the previous hack don't have the 2nd feature

Lmao, where your skill, I can do it without hack

**Comment by iBLiSSIN** [comment]
> Also please resend the discord invition link

I send it for u on discord

**Comment by IceHacks** [comment]
The second part is of interest.

**Comment by TylerWhat** [comment]
@iBLiSSIN , can u send out the download resources for 3.0.   thanks
